src/Feed.js

```diff
@@ -5,27 +5,24 @@ import ImageIcon from "@mui/icons-material/Image";
 import SubscriptionsIcon from "@mui/icons-material/Subscriptions";
 import {
   addDoc,
-  collection,
-  onSnapshot,
-  orderBy,
-  serverTimestamp,
-  getDocs,
+  collection, getDocs, orderBy,
+  serverTimestamp
 } from "firebase/firestore";
-import React, { useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import { useSelector } from "react-redux";
+import { selectUser } from "./features/userSlice";
 import "./Feed.css";
 import { db } from "./firebase";
 import InputOption from "./InputOption";
 import Post from "./Post";
-import { selectUser } from "./features/userSlice";
 
 /* 
   Firebase v9 (Modular docs)
   https://firebase.google.com/docs/firestore/manage-data/add-data#add_a_document
 */
 
 function Feed() {
-  const user = useSelector(selectUser);
+  const {user} = useSelector(selectUser);
```

Comment:
You were required to destructure the user key from the selector object of redux store.

Reply:
@AmanRehan please see the changes made, other unnecessary imports have also been removed.